toy_settings/django_back_end/queries.py

```diff
@@ -24,8 +24,16 @@ def events_for_key(self, key: str) -> list[events.Event]:
 
     def current_value(self, key: str) -> str | None:
         """Get the current value of a setting."""
-        return projections.current_settings(self.events_for_key(key)).get(key)
+        setting = projections.current_settings(self.events_for_key(key)).get(key)
+        if setting is None:
+            return None
+        else:
+            return setting.value
 
     def all_settings(self) -> dict[str, str]:
         """Get the current value of all settings."""
-        return projections.current_settings(self._events())
+        return {
+            key: setting.value
+            for key, setting in projections.current_settings(self._events()).items()
+            if setting.value is not None
+        }
```

Comment:
Thought: would we ever want a setting to have the value `None`?

Reply:
All settings are strings, at the moment